sdf.md

```diff
@@ -1567,7 +1567,7 @@ thus the common qualities), see {{data-qualities}}, additional qualities are sho
 
 The `sdfAction` keyword denotes a group of zero or more Action definitions.
 
-Actions are used to model commands and methods which are invoked. Actions have parameter data that are supplied upon invocation.
+Actions are used to model commands and methods which are invoked. Actions may have parameter data that are supplied upon invocation and output data that is provided as a result of the invocation of the action.
```

Comment:
Maybe "direct result" (as opposed to through an action object)?

Reply:
or maybe "output data or action object"?